python/examples/tcp_client.py

```diff
@@ -38,20 +42,24 @@
 
     # Open the output file if logging was requested.
     if options.output is not None:
-        p1i_path = os.path.splitext(options.output)[0] + '.p1i'
-        if os.path.exists(p1i_path):
-            os.remove(p1i_path)
+        if options.format == 'p1log':
+            p1i_path = os.path.splitext(options.output)[0] + '.p1i'
```

Comment:
its a bit weird that the extension and the input arg are different....perhaps we should support both p1log and p1i?